packages/flutter_goldens_client/lib/skia_client.dart

```diff
@@ -0,0 +1,215 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io' as io;
+
+import 'package:file/file.dart';
+import 'package:path/path.dart' as path;
+import 'package:platform/platform.dart';
+import 'package:process/process.dart';
+
+import 'package:flutter_goldens_client/client.dart';
+
+// If you are here trying to figure out how to use golden files in the Flutter
+// repo itself, consider reading this wiki page:
+// https://github.com/flutter/flutter/wiki/Writing-a-golden-file-test-for-package%3Aflutter
+
+// TODO(Piinks): This file will replace ./client.dart when transition to Skia
+// Gold testing is complete
+const String _kGoldctlKey = 'GOLDCTL';
+const String _kServiceAccountKey = 'GOLD_SERVICE_ACCOUNT';
+
+/// An extension of the [GoldensClient] class that interfaces with Skia Gold
+/// for golden file testing
+class SkiaGoldClient extends GoldensClient {
+
+  SkiaGoldClient() : super();
```

Comment:
Why the constructor split?

Reply:
I think the constructor split is a bit more relevant now. Otherwise it overrides the assignment from super.